lib/services/linux.dart

```diff
@@ -592,10 +592,16 @@ class Linux {
       newEnvironment.distribution = DISTROS.UBUNTU;
     } else if (lines[0].toLowerCase().contains("debian")) {
       newEnvironment.distribution = DISTROS.DEBIAN;
+    } else if (lines[0].toLowerCase().contains("arch")) {
```

Comment:
Great! But how to handle the package management? The after Installation screen e.g.? We need much more development in the linux.dart file that the linux assistant works with arch distros